psalm-baseline.xml

```diff
@@ -1,5 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<files psalm-version="4.x-dev@5108834088c1d8cae3698df6ef6bf15fe6e76c53">
+<files psalm-version="4.30.0@d0bc6e25d89f649e4f36a534f330f8bb4643dd69">
+  <file src="examples/aggregate.php">
```

Comment:
I assume this files was auto-generated and there's no need to review.

Reply:
That's correct.